internal/olm/client/client.go

```diff
@@ -48,6 +48,22 @@ var ErrOLMNotInstalled = errors.New("no existing installation found")
 
 var Scheme = scheme.Scheme
 
+// custom error struct to capture deployment errors
+// while verifying CSV installs.
+type deploymentError struct {
+	depName string
```

Comment:
```suggestion
	name string
```
imo: name is enough. see that it would be `deploymentError.name`